app/auto-eligibility/lib/process-waiting-list.js

```diff
@@ -1,8 +1,20 @@
+const applyServiceUri = require('../config').applyServiceUri
+const updateAccessGranted = require('../../auto-eligibility/processing/update-access-granted')
+const { sendApplyGuidanceEmail } = require('../../auto-eligibility/email')
+
 const processWaitingList = async (upperlimit) => {
   console.log(`Executing process waiting list with limit of ${upperlimit}.`)
-  // logic to handle waiting list functionality
-  // todo https://dev.azure.com/defragovuk/DEFRA-FFC/_workitems/edit/135655
-  // todo https://dev.azure.com/defragovuk/DEFRA-FFC/_workitems/edit/135656
+  try {
+    const result = await updateAccessGranted()
+    const updatedRows = result[0]
+    const rowsUpdated = result[1]
+    console.log(`${updatedRows} farmers moved from the waiting list.`)
+    rowsUpdated.forEach(farmer => {
+      sendApplyGuidanceEmail(farmer.business_email, applyServiceUri)
```

Comment:
would be nice to send a bulk email instead of looking through and making N amount of API calls (maximum 50)